airbyte-workers/src/main/java/io/airbyte/workers/WorkerUtils.java

```diff
@@ -112,4 +115,18 @@ public static StandardTargetConfig syncToTargetConfig(StandardSyncInput sync) {
         .withState(sync.getState());
   }
 
+  // todo (cgardens) - there are 2 sources of truth for job path. we need to reduce this down to one,
+  // once we are fully on temporal.
+  public static Path getJobRoot(Path workspaceRoot, IntegrationLauncherConfig launcherConfig) {
```

Comment:
good call. I think you can replace the code in `WorkerRun` on line 52 with this too. gives us a better chance of making sure we get the paths right in temporal and outside of it.